src/components/Header/Header.js

```diff
@@ -97,6 +97,29 @@ const NavAuthUser = props => {
                     </Button>
                   </List>
                 </div>
+              ) : rol === "mentor" ? (
+                <div>
+                  <List>
+                    <Button
+                      tabIndex="-1"
+                      color="inherit"
+                      component={Link}
+                      to="/mentorshome"
+                    >
+                      Set state
```

Comment:
What about "Add status" or "Update status" sounds less "React-minded" 😜

Reply:
That is very true. I have changed it. 👍